tests/auto/tst_declarativebookmarkmodel/tst_declarativebookmarkmodel.pro

```diff
@@ -1,15 +1,13 @@
 TARGET = tst_declarativebookmarkmodel
 
-QT += quick concurrent
+QT += network concurrent
```

Comment:
`src/bookmarks/iconfetcher.cpp` (included by `src/bookmarks/bookmarks.pri`) needs it. I think it makes sense to refactor the tests not to depend on iconfetcher, but probably in a separate PR.